pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -78,6 +78,8 @@ public static class Helix {
     public static final String LEAD_CONTROLLER_RESOURCE_ENABLED_KEY = "RESOURCE_ENABLED";
 
     public static final String ENABLE_CASE_INSENSITIVE_KEY = "enable.case.insensitive";
+    public static final String CONFIG_OF_ALLOW_TABLE_NAME_DOTS = "allow.table.name.dots";
```

Comment:
Per the discussion, suggest changing it to `ALLOW_TABLE_NAME_WITH_DATABASE`, and only allow one dot if enabled

Reply:
Changed.